sec_edgar_downloader/rate_limiter.py

```diff
@@ -0,0 +1,32 @@
+import typing
+
+from datetime import datetime
+
+PWN = typing.TypeVar('PWN')
```

Comment:
Typing is not yet officially supported. Let's hold this off until I get typing into the rest of the package (I am working on it)!